packages/terra-dropdown-button/src/DropdownButton.jsx

```diff
@@ -57,29 +57,57 @@ class DropdownButton extends React.Component {
     this.handleDropdownRequestClose = this.handleDropdownRequestClose.bind(this);
     this.handleKeyDown = this.handleKeyDown.bind(this);
     this.handleKeyUp = this.handleKeyUp.bind(this);
+    this.setButtonNode = this.setButtonNode.bind(this);
+    this.getButtonNode = this.getButtonNode.bind(this);
+    this.setListNode = this.setListNode.bind(this);
+    this.state = { isOpen: false, isActive: false, isKeyboardEvent: false };
```

Comment:
I don't understand what `isKeyboardEvent` signifies in context of this component. What does it mean? Could it be something like `keyboardAccessed`, `keyPressed` or `openedViaKeyboard` (quite long)?

Reply:
changed to : openedViaKeyboard